### PR TITLE
work around application with multibyte file name and Python 2

### PR DESCRIPTION
**Pull request by nishimotz**
### Link to issue number:

#9583

### Summary of the issue:

Application with multibyte file name, such as "テストアプリケーション.exe" raises error.

### Description of how this pull request fixes the issue:

If Python 2 is used, make sure importlib.import_module uses bytes rather than unicode.

### Testing performed:

not yet.

### Known issues with pull request:

Haven't identified why this regression has happened. 

### Change log entry:

Section: Bug fixes

### CODE REVIEW

**Comment by dingpengyu** [comment]
hi nishimotz
The following error occurred when appveyor was built:
Failure: AppVeyor build failed
Can you fix this build issue?
thank

File: source/appModuleHandler.py [hunk 1]
```diff
@@ -171,7 +171,9 @@ def fetchAppModule(processID,appName):
 	# First, check whether the module exists.
 	# We need to do this separately because even though an ImportError is raised when a module can't be found, it might also be raised for other reasons.
 	# Python 2.x can't properly handle unicode module names, so convert them.
-	modName = appName.encode("mbcs")
+	modName = appName
+	if sys.version_info.major == 2 and not isinstance(modName, bytes):
```
Comment (leonardder) [inline source/appModuleHandler.py]: I think this should be sufficient

```suggestion
	if not isinstance(modName, bytes):
```

**Review by LeonarddeR — APPROVED** [review]
Tested and working as expected.

**Comment by nishimotz** [comment]
As explained in #9583, Ctrl+NVDA+F1 does not work.
I am trying a different approach to the issue, which is better as the preparation for Python 3, in my opinion.

**Comment by nishimotz** [comment]
Ctrl+NVDA+F1 (script_reportAppModuleInfo) works with pr9590-17289,9bfdf943 as expected.
I have tried another approach. However, Windows Defender falsely detected the binary as malware, so I gave up for now.

**Comment by leonardder** [comment]
I'm afraid this is getting a huge mess with regard to what type/encoding is used.

I think we should ensure that appModuleHandler.getAppNameFromProcessID should return unicode at all times.

This means:

* In the case of `if processID==NVDAProcessID`, a unicode literal should be added to the strings below
* When querying the app module for the appname, the appname should no longer be replaced by the encoded version.
* getAppNameFromHost should return unicode as well

**Comment by leonardder** [comment]
Closing in favour of #9635

**Pull request closed without merging**